deploy/base/operator.yaml

```diff
@@ -33,6 +33,8 @@ spec:
               value: non-root-enabler
             - name: RELATED_IMAGE_SELINUXD
               value: quay.io/jaosorior/selinuxd
+            - name: RELATED_IMAGE_LOG_ENRICHER
+              value: paulinhu/kube-audit-log-enricher
```

Comment:
Wondering if we want to put the sources from the dependency project (https://github.com/pjbgf/kube-audit-log-enricher) into the operator. Any thoughts on that? :)

Reply:
Let's make this a topic for next project call. We need to make a similar decision for selinuxd.

Reply:
Just added it to the monthly meeting - but happy to discuss it Thursday if we have the time.